docs/version-history/v23-version.hist

```diff
@@ -1,3 +1,7 @@
 *** 23.0.21 bugs
 
 *** 23.0.20 bugs
+
+- Updated :tool:`condor_upgrade_check` to test for use for PASSWORD
```

Comment:
I don't think that :tool: is available in 23.0. @ColeBollig would know for sure.

Reply:
It is not